Source/Scene/LabelCollection.js

```diff
@@ -329,6 +353,10 @@ define([
         this._totalGlyphCount = 0;
         this._resolutionScale = undefined;
 
+        this._renderedTileList = [];
```

Comment:
Here and throughout (e.g., `tileRenderedEvent`), is this really "rendered" or "newly visible?"